### PR TITLE
Fix PR-based CI to account for files that have been deleted

### PR DESCRIPTION
**Pull request by sgibson91**
### Summary

When running CI in PRs, CI will fail with a `FileNotFoundError` if the PR deletes a file. This PR adds a `try/except` block to try to circumvent this. It could probably be handled more elegantly with something in the JSON packet from `pull_files.py`, but this should work for now.

### List of changes proposed in this PR (pull-request)

* `try/except` blocks added to `lorem-ipsums.py` and `no-bad-latin.py`

### What should a reviewer concentrate their feedback on?

- [ ] 🚨 Please check my indentation! Editing on GitHub web interface is hard 🙁 
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: tests/lorem-ipsums.py [hunk 1]
```diff
@@ -53,13 +53,17 @@ def check_changed_files(pr_num, bad_phrase=BAD_PHRASE):
...
-        text = f.read()
-        text = remove_comments(text)
-        if bad_phrase in text.lower():
-            failed.append(filename.name)
+	    try:
+            f = open(
+                os.path.join(ABSOLUTE_HERE, filename), encoding="utf8", errors="ignore"
+            )
+            text = f.read()
+            text = remove_comments(text)
```
Comment (malvikasharan) [inline tests/lorem-ipsums.py:61]:
```suggestion
            f = open(
            os.path.join(ABSOLUTE_HERE, filename), encoding="utf8", errors="ignore"
            )
            text = f.read()
            text = remove_comments(text)
```

Comment (malvikasharan) [reply tests/lorem-ipsums.py:61]: removing mix of tab and space error

**Comment by malvikasharan** [comment]
Ok. Let's merge this and I will pull the change in the other PR.

**Comment by sgibson91** [comment]
Sorry fixing it up was such a pain 😅

**Comment by malvikasharan** [comment]
I think my Atom needs to be updated. This was not supposed to be challengeing :D Thank you for the fix.

**Pull request merged**